procedures/release-duty/index.rst

```diff
@@ -340,30 +331,31 @@ No, after b1, there isn't signoff from RelMan on DevEdition builds. QA only veri
 weeks. With the exception of b1, and assuming all the tasks complete as expected, the DevEdition builds should be
 shipped at the same time as we receive signoff for the corresponding desktop builds.
 
-7. *How should I inform the ReleaseDuty team of recent changes in automation that may impact an upcoming release?*
-
-You can mention it to the current ReleaseDuty folks in the #releaseduty channel. Please also add it to the upcoming
-release in the ../releases/FUTURE/ dir. See `future release support <../releases/FUTURE/README.md>`__ for more details.
```

Comment:
If we keep this, we would change the future releases link to the changelog.
Historically we've tried to also be around for beta1 and the rc/release if we've made a major change in release automation that might go south on release day.